apps/microsoft-teams/frontend/src/components/config/ContentTypeSelectionModal/ContentTypeSelectionModal.tsx

```diff
@@ -44,14 +72,15 @@ const ContentTypeSelectionModal = (props: Props) => {
       );
     }
 
-    if (contentTypes.length) {
```

Comment:
renamed `contentTypes` to `filteredContentTypes` since we no longer using just the exact set that is provided via `useGetContentTypes() & ContentType context`, now we want to possibly render a subset of that data.  Open to suggestions of better name than `filtered...`